src/components/BulkUploadListItem.tsx

```diff
@@ -19,7 +20,7 @@ interface BulkUploadListItemProps {
 
 export const BulkUploadListItem = ({ upload }: BulkUploadListItemProps) => {
   const {
-    id, fileName, status, createdAt,
+    id, fileName, fileSize, status, createdAt,
```

Comment:
w00t file size is back!

Rather than having all these commits with thrash would it be easy to reorder / smoosh / remove things so that it's just introduced once?